sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -44,39 +47,6 @@
 * Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also added `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
 * Removed `System.exit()` calls from `ImplementationBridgeHelpers`. - See [PR 39387](https://github.com/Azure/azure-sdk-for-java/pull/39387)
 
-### 4.53.4-hotfix (2024-03-15)
```

Comment:
Why the reordering of changelog? This should happen in a separate PR if it is necessary. Please revert.

Reply:
Faulty merge - fixed in next iteration.